components/site/src/sitemap.rs

```diff
@@ -92,24 +92,19 @@ pub fn find_entries<'a>(
         if !taxonomy.kind.render {
             continue;
         }
-        let name = &taxonomy.kind.name;
-        entries.insert(SitemapEntry::new(Cow::Owned(config.make_permalink(name)), &None));
+        entries.insert(SitemapEntry::new(Cow::Borrowed(&taxonomy.permalink), &None));
 
         for item in &taxonomy.items {
-            entries.insert(SitemapEntry::new(
-                Cow::Owned(config.make_permalink(&format!("{}/{}", name, item.slug))),
-                &None,
-            ));
+            entries.insert(SitemapEntry::new(Cow::Borrowed(&item.permalink), &None));
 
             if taxonomy.kind.is_paginated() {
                 let number_pagers = (item.pages.len() as f64
                     / taxonomy.kind.paginate_by.unwrap() as f64)
                     .ceil() as isize;
                 for i in 1..=number_pagers {
                     let permalink = config.make_permalink(&format!(
-                        "{}/{}/{}/{}",
-                        name,
-                        item.slug,
+                        "{}{}/{}",
+                        item.path,
```

Comment:
are we ensuring it ends with a trailing slash upstream already?

Reply:
Yes, `config.make_permalink` ensures it.
I've now added the slash nevertheless just for the consistency.